### ISSUE TITLE
[Documentation] The changelog is not mentioning a breaking change.

### ISSUE THREAD

**Issue by noraj** [sweetalert2/sweetalert2#1608]
## Description

See https://stackoverflow.com/questions/56416187/sweetalert2-typeerror-this-is-undefined-after-v8-upgrade/

**TL;DR**: The changelog is not mentioning a breaking change.

Before (I don't know exactly when since the changelog do not talk about it) calling `Swal()` was enough to launch the popup, now (I don't know since when) `Swal.fire()` is mandatory. The issue is that the Changelog don't talk about that.

More information in the StackOverflow post.

## Current behavior

Issue in the documentation not in the code!

## Expected behavior

Issue in the documentation not in the code!

## Live demo

Issue in the documentation not in the code!

**Comment by gverni** [comment]
Hi @noraj the breaking change is clearly mentioned in the [release note for version 8.x](https://github.com/sweetalert2/sweetalert2/releases/tag/v8.0.0). And this is linked on the top of the README (look for "Upgrading from v7.x to v8.x?"). This is also reported in the changelog pointed in the stackoverflow article you mentioned: you need to expand the commit f581352 "BREAKING CHANGE: remove withNoNewKeyword enhancer". I'll update the stackoverflow article accordingly.

**Comment by noraj** [comment]
Hi @gverni when I did my upgrade I directly went to `CHANGELOG.md` and saw the [8.0.0 (2019-01-19)](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md#800-2019-01-19) with a lot of `BREAKING CHANGE`. 

I only targeted `CHANGELOG.md` because this is the file to target on every project and when you have several dependencies to review you can't spend time to find f the info is elsewhere.

But that's true the information is on the release page [`tag/v8.0.0`](https://github.com/sweetalert2/sweetalert2/releases/tag/v8.0.0). I didn't see `Upgrading from v7.x to v8.x` in the README.

So the problem is that `CHANGELOG.md` is not reflecting all changes that are in https://github.com/sweetalert2/sweetalert2/releases. Some project don't bother with both and just put links to tags in the CHANGELOG.md file (I don't think it is a thing to do but a least you can't miss something.

## TL;DR

Release GitHub page

![](https://i.imgur.com/T7dxKC7.png)

**Versus**

CHANGELOG.md

![](https://i.imgur.com/C2nLy7x.png)

**Comment by noraj** [comment]
@gverni 

- `Breaking change # 2 - Bower support has been discontinued`
- `Breaking change # 3 - Deprecated methods were removed`
- `Breaking change # 4 - Deprecated parameters were removed`
- `Breaking change # 5 - Android Browser is no longer supported`

Are in the `CHANGELOG.md` but not `Breaking change # 1 - swal(...) -> Swal.fire(...)`

**Comment by noraj** [comment]
I made a PR #1608

**Comment by gverni** [comment]
> Are in the CHANGELOG.md but not Breaking change # 1 - swal(...) -> Swal.fire(...)
 
@noraj as I reported already in my comment, the breaking change for `Swal.fire()` is already in commit f581352 under title "BREAKING CHANGE: remove withNoNewKeyword enhancer". I agree that this does not explicitly mention the `fire()` method, but this can be seen inside the commit message. 

I let @limonte comment on the PR for the changelog. Keep in mind that the changelog is automatically generated using the commit title, and there is no commit with an explicit title referring to `Swal.fire()`. If we have to change the changelog, then I would rather add the link to the release note.

**Comment by limonte** [comment]
Thank you @noraj and @gverni for the constructive discussion, the PR is merged now :+1:
